### PR TITLE
add badges

### PR DESCRIPTION
**Pull request by giovp**

### CODE REVIEW

**Comment by giovp** [comment]
remove pyupgrade flag in ruff as obsolete see https://github.com/astral-sh/ruff/blob/main/BREAKING_CHANGES.md#00268

we could ignore it via `UP006` and `UP007`, @michalk8 wdyt?

**Comment by giovp** [comment]
please check both readme and landing page of docs to see whether you like the badges (the pre-commmit one will be updated once merged in main)

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -7,18 +7,18 @@ default_stages:
...
         files: ^src
   - repo: https://github.com/psf/black
-    rev: 23.1.0
+    rev: 23.3.0
     hooks:
       - id: black
         additional_dependencies: [toml]
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.0.0-alpha.6
+    rev: v3.0.0-alpha.9-for-vscode
```
Comment (MUCDK) [inline .pre-commit-config.yaml:21]: what does `for-vscode` mean?

**Review by MUCDK — APPROVED** [review]
Thanks Giovanni!

**Review by michalk8 — APPROVED** [review]

**Pull request merged**